contents/uix/accessibility.md

```diff
@@ -0,0 +1,137 @@
+<frontmatter>
+  title: Accessibility
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Accessibility
+
+Author(s): [Monika Manuela Hengki](https://github.com/monmanuela)
+
+## What is accessibility?
+
+> Accessibility is the practice of making your websites usable by as many people as possible — we traditionally think of this as being about people with disabilities, but really it also benefits other groups such as those using mobile devices, or those with slow network connections.
+>
+> Taken from [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/What_is_accessibility)
+
+Accessibility enables users, regardless or their ability or circumstances, to perceive, interact and navigate through a website without barriers. Just like how buildings have wheelchair ramps or elevators to help wheelchair-bound people enter a building, a website can have some nifty tricks to help users with disabilities navigate through it.
+
+## Why is accessibility important?
+
+##### For the developer
+1. **Accessibility improves everyone's UX**. Accessibility practices are good design practices in general, so improving accessibility also makes your website more usable by other groups such as mobile phone users, those on a low network speed, etc.
+2. **Accessibility is a right**. Caring about accessibility demonstrates good ethics/morals, which improves your public image.
+3. **Accessibility improves SEO**. Semantic HTML (which improves accessibility) also improves SEO, making your website more findable and marketable to new users.
+4. **Accessibility helps you increase your user base**. According to the [World Health Organization world report on disability](https://www.who.int/disabilities/world_report/2011/report/en/), about 15% of the world's population live with some form of disability. That makes up more than 1 billion people. It is a significant population of users you can reach just by improving accessibility.
+5. **Accessibility is part of the law in some places**. In [some countries](https://www.w3.org/WAI/policies/), abiding by accessibility guidelines is mandatory. Be careful not to break the law!
+
+##### For the user
+Accessibility is important for a lot of people with disabilities to access the Web. There are diverse kinds of disabilities, including:
```

Comment:
'for a lot of people with disabilities` - rephrase? (I find it hard to read)

Reply:
'diverse kinds of disabilities' - diverse means showing a great deal of variety? Sounds odd to use it to describe disabilities